bigtable-client-core-parent/bigtable-v1-protos/README.md

```diff
@@ -0,0 +1,76 @@
+# Bigtable Protos
```

Comment:
Perhaps bigtable-legacy-protos might be a better name for this? Or alternatively we could rename bigtable-protos to bigtable-v2-protos? I think either would be slightly less confusing for new people in the codebase.